lib/need_for_type/states/score.rb

```diff
@@ -5,20 +5,22 @@
 module NeedForType::States
   class Score < State
 
-    def initialize(display_window, time, wpm, accuracy)
+    def initialize(display_window, time, wpm, accuracy, difficulty)
       super(display_window)
+      @difficulty = difficulty
       @time = time
       @wpm = wpm
-      @accuracy = accuracy 
+      @accuracy = accuracy
+      @option = 0
     end
 
     def update
-      @display_window.render_score(@time, @wpm, @accuracy)
+      @display_window.render_score(@time, @wpm, @accuracy, @option)
 
-      input = @display_window.get_input
-
-      if input == Curses::Key::ENTER || input == 10
-        return NeedForType::States::Menu.new(@display_window)
+      input_worker(3) do
+        exit if @option == 2
+        return NeedForType::States::Game.new(@display_window, @difficulty) if @option == 0
+        return NeedForType::States::Menu.new(@display_window) if @option == 1
```

Comment:
I would use a `case` statement here, I think it make code more readable.
What do you think ?